db/schemas/mysql/3.01-3.02/110-create-drop_index_if_exists-proc.sql

```diff
@@ -0,0 +1,15 @@
+DROP PROCEDURE IF EXISTS smt.drop_index_if_exists;
+
+CREATE PROCEDURE smt.drop_index_if_exists(in theTable varchar(128), in theIndexName varchar(128) )
+BEGIN
+ IF((SELECT COUNT(*) AS index_exists
```

Comment:
This fails on schema upgrade.
From what I googled out, there is IF and IF() where it seems to me you are using syntax of IF, but calling it as IF(). Can be that the culprit?
Actual error is:
`ERROR 1064 (42000) at line 4: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 8
3.01-3.02/110-create-drop_index_if_exists-proc.sql`